esphome/components/media_player/__init__.py

```diff
@@ -103,6 +109,25 @@ async def register_media_player(var, config):
 MEDIA_PLAYER_ACTION_SCHEMA = maybe_simple_id({cv.GenerateID(): cv.use_id(MediaPlayer)})
 
 
+@automation.register_action(
+    "media_player.play_media",
+    PlayMediaAction,
+    cv.maybe_simple_value(
+        {
+            cv.GenerateID(): cv.use_id(MediaPlayer),
+            cv.Required(CONF_MEDIA_URL): cv.templatable(cv.string_strict),
```

Comment:
`cv.url` might be better here for validation

```suggestion
            cv.Required(CONF_MEDIA_URL): cv.templatable(cv.url),
```